cobalt/renderer/backend/graphics_system_test.cc

```diff
@@ -57,10 +57,10 @@ TEST(GraphicsSystemTest, FLAKY_GraphicsSystemCanBeInitializedOften) {
                << "ms per initialization.";
 
   // Graphics system initializations should not take more than the maximum of
-  // 200ms or three times as long as the time we just measured.
+  // 250ms or three times as long as the time we just measured.
```

Comment:
this is ~ 3.5 x now. upto you to change it

Reply:
same for the comment below

Reply:
I think you are misreading the expression. It's maximum of either 3 times per the initialization test, OR 250 milliseconds. It was added here go/cobalt-cl/152480